glue/core/data.py

```diff
@@ -2057,3 +2060,169 @@ def pixel_label(i, ndim):
     if 1 <= ndim <= 3:
         label += " [{0}]".format('xyz'[ndim - 1 - i])
     return label
+
+
+class RegionData(Data):
+    """
+    A glue Data object for storing data that is associated with a region.
```

Comment:
Yes -- I'll include this in the new version in your new suggested location.

Reply:
I have moved some of the key logic around displaying regions from the Viewer (where it would have to be repeated) into the RegionData object. This docstring now mentions why to use RegionData.